topic-operator/pom.xml

```diff
@@ -214,6 +218,15 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <configuration>
+                    <systemPropertyVariables>
+                        <vertx.logger-delegate-factory-class-name>io.strimzi.operator.topic.vertx.OverrideBlockedThreadCheckerLoggerDelegateFactory</vertx.logger-delegate-factory-class-name>
+                    </systemPropertyVariables>
+                </configuration>
+            </plugin>
```

Comment:
Added this system property as part of my Rube Goldberg machine to override the BlockedThreadChecker's logger, as that's the only way to hook into it for unit testing - maybe the unit test isn't worth the overhead?

Reply:
Done :)